pkg/apis/ceph.rook.io/v1/object.go

```diff
@@ -76,6 +76,9 @@ func ValidateObjectSpec(gs *CephObjectStore) error {
 	if gs.Namespace == "" {
 		return errors.New("missing namespace")
 	}
+	if len(gs.Name) > 38 {
```

Comment:
Please add a comment here that explains why 38 is the critical number, and maybe show a formula for how it was decided.

Reply:
Added